guide/contribute/case-studies.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: guide
+title: Case studies
+description: How to contribute case studies to the guide.
+nav_order: 10
+parent: Contribute to guide
+permalink: /guide/contribute/case-studies/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/contribute/contribute-preview.jpg
+---
+
+{% include picture.html
+   image = "assets/images/guide/contribute/casestudies-header.jpg"
+   retina = "assets/images/guide/contribute/contribute/casestudies-header@2X.jpg"
+   mobile = "assets/images/guide/contribute/casestudies-header-mobile.jpg"
+   mobileRetina = "assets/images/guide/contribute/casestudies-header-mobile@2X.jpg"
+   customImage = "assets/images/guide/contribute/casestudy-contribute-custom-image.jpg
```

Comment:
Not sure where this `customImage` attribute comes from, but this does not exist in `picture.html`. You can remove it.